flagsmith-on-aks/README.md

```diff
@@ -0,0 +1,119 @@
+# Flagsmith AKS + Helm Terraform module
+
+## Prerequisites
+
+### Tools
+
+Install the prerequisites:
+
+```sh
+brew update && brew install azure-cli kubernetes-cli helm terraform
+```
+
+Authorize your Azure CLI:
+
+```sh
+az login
+```
+
+## Deployment
+
+### Inputs
```

Comment:
It would be good to indicate which of these are required. For example, it feels as though the `flagsmith_on_flagsmith_api_key` shouldn't be required.